pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.smartling.marketo</groupId>
     <artifactId>marketo-sdk</artifactId>
-    <version>3.1.27</version>
+    <version>3.1.27-SNAPSHOT</version>
```

Comment:
Why did you decide to use snapshot version? And why not 3.1.28-SNAPSHOT?

Reply:
forgot increase minor version. I must use SNAPSHOT in branches, otherwise the artifact will be uploaded to the repository only one time